frontend/src/components/compound/SpaceListing.tsx

```diff
@@ -0,0 +1,99 @@
+import SpaceListingService from "../../lib/services/Space-Listing/service";
+
+import { UserListingType } from "../../lib/services/User-Listing/types";
+import TwitterIcon from "@mui/icons-material/Twitter";
+import ContactMe from "../helper/ContactMe";
+import { SpaceListing } from "../../lib/services/Space-Listing/types";
+import PeopleOutlineIcon from "@mui/icons-material/PeopleOutline";
+import LocationOnOutlinedIcon from "@mui/icons-material/LocationOnOutlined";
+import DescriptionBox from "./DescriptionBox";
+
+//https://pbs.twimg.com/profile_images/1387824030602780673/CqiWzrma_400x400.jpg
+
+export default function SpaceListingCard({
+  SpaceData,
+}: {
+  SpaceData: SpaceListing;
+}) {
+  const UserData = SpaceData;
```

Comment:
FYI be careful that this PR does not add back the `UserData` usage that we are removing in https://github.com/fractal-bootcamp/directoryNY/pull/35